ide/options.editor/src/org/netbeans/modules/options/editor/completion/Bundle.properties

```diff
@@ -51,3 +51,4 @@ AN_Javadoc_Next_To_CC=Documentation Next To Completion Checkbox
 AD_Javadoc_Next_To_CC=Display Documentation Next To Completion
 GeneralCompletionOptionsPanel.AccessibleContext.accessibleName=General Completion Options
 GeneralCompletionOptionsPanel.AccessibleContext.accessibleDescription=Settings for various completion settings
+GeneralCompletionOptionsPanel.cbDisplayParameterTooltip.text=Display Parameter &Tooltip
```

Comment:
regarding the UI text: might be better to change it to "Auto Popup Parameter &Tooltip" so that it fits to the other two check boxes which follow the same pattern.